ui/DesignSystem/Component/ProjectList.svelte

```diff
@@ -41,6 +41,8 @@
       class="project-card">
       <ProjectCard
         title={project.metadata.name}
+        projectId={project.id}
+        {registrarId}
```

Comment:
Passing these through deep in the component tree, seem sub-optimal and is in relation to the routing comment above in `ProjectCard`.

Reply:
See https://github.com/radicle-dev/radicle-upstream/commit/cb15f7cd38bc77caf8decdc0871336e2163e1433.